addons/web/static/src/views/fields/properties/properties_field.scss

```diff
@@ -82,7 +82,7 @@
 
 .o_property_field_popover {
     // put the popover behind datetime component
-    z-index: 1000 !important;
+    z-index: 1100 !important;
```

Comment:
Have you tried with the actual Bootstrap's variable dedicated to popovers : `$zindex-popover` ?

Reply:
Works like charm too, thanks 🙂